### ISSUE TITLE
Wire.h: No such file or directory

### ISSUE THREAD

**Issue by davegutz** [beefbowl/Spark_Adafruit_LEDBackpack#2]
Veryfing code using this library in Particle Build fails with this error:

```
Adafruit-LEDBackpack/Adafruit-LEDBackpack.h:25:19: fatal error: Wire.h: No such file or directory
  #include <Wire.h>
                   ^
My application.cpp code looks like this:

#include "application.h"
//#include "Adafruit_LEDBackpack.h"
// This #include statement was automatically added by the Particle IDE.
#include "Adafruit-LEDBackpack/Adafruit-LEDBackpack.h"
#include <math.h>
...

Where I ported from old supposedly working thermostat hackathon code from early in 2014 into the Web IDE and I had to replace the include Adafruit with what's shown to get it to compile.    If you know what's going on I'd appreciate a word or two.  Thanks.
```

**Comment by xitit** [comment]
I looked, I couldn't find a reference to the #include <Wire.h> statement...  Does it need to be committed to Particle somehow?

**Comment by davegutz** [comment]
I cannot reproduce the problem now (actually, I could if I ran the original code).     I started over from scratch, cutting and pasting to rebuild the calling program (not the LED code).   And it compiles and runs properly.   I could not tell  you what the problem was without an incremental build investigation.   It's something weird I'm sure.   I was getting nonsense compiler messages at one point due to .ino preprocessor / C++ compiler issues and was getting very confused.  Thanks.
